management/management.go

```diff
@@ -82,9 +82,17 @@ type Management struct {
 // New creates a new Auth0 Management client by authenticating using the
 // supplied client id and secret.
 func New(domain, clientID, clientSecret string, options ...apiOption) (*Management, error) {
+	parsedDomain, err := url.Parse(domain)
```

Comment:
If we rename to url then this would make more sense being `u, err := url.Parse(domain)`. Do you agree?